test/tests/conftest.py

```diff
@@ -3,161 +3,214 @@
 import subprocess
 import testinfra
 
-local_host = testinfra.get_host('local://')
+local_host = testinfra.get_host("local://")
 check_output = local_host.check_output
 
-TAIL_DEV_NULL='tail -f /dev/null'
+TAIL_DEV_NULL = "tail -f /dev/null"
+
 
 @pytest.fixture()
 def run_and_stream_command_output():
     def run_and_stream_command_output_inner(command, verbose=False):
         print("Running", command)
         build_env = os.environ.copy()
-        build_env['PIHOLE_DOCKER_TAG'] = version
-        build_result = subprocess.Popen(command.split(), env=build_env, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
-                                        bufsize=1, universal_newlines=True)
+        build_env["PIHOLE_DOCKER_TAG"] = version
+        build_result = subprocess.Popen(
+            command.split(),
+            env=build_env,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            bufsize=1,
+            universal_newlines=True,
+        )
         if verbose:
             while build_result.poll() is None:
                 for line in build_result.stdout:
-                    print(line, end='')
+                    print(line, end="")
         build_result.wait()
         if build_result.returncode != 0:
-            print(f'       [i] Error running: {command}')
+            print(f"       [i] Error running: {command}")
             print(build_result.stderr)
+
     return run_and_stream_command_output_inner
 
+
 @pytest.fixture()
 def args_volumes():
-    return '-v /dev/null:/etc/pihole/adlists.list'
+    return "-v /dev/null:/etc/pihole/adlists.list"
+
 
 @pytest.fixture()
 def args_env():
     return '-e FTLCONF_LOCAL_IPV4="127.0.0.1"'
 
+
 @pytest.fixture()
 def args(args_volumes, args_env):
     return "{} {}".format(args_volumes, args_env)
 
+
 @pytest.fixture()
 def test_args():
-    ''' test override fixture to provide arguments separate from our core args '''
-    return ''
+    """test override fixture to provide arguments separate from our core args"""
```

Comment:
I saw many strings like this, where spaces were trimmed (begin and end of the string).

Was this intentional?
Apparently this will change nothing (maybe just the output format), but I think it's better to ask.

Reply:
I believe trimming strings only happens in these doc-blocks. Trimming a regular sting could have side effects, but Black is smart enough to figure out these are just doc-blocks, and so doesn't affect the code.

Reply:
Yeah... I updated my comment at the same time you were answering.
I'm not used to this Python comment format.

Reply:
I found the black rules for these:

> Black also processes docstrings. Firstly the indentation of docstrings is corrected for both quotations and the text within, although relative indentation in the text is preserved. Superfluous trailing whitespace on each line and unnecessary new lines at the end of the docstring are removed. All leading tabs are converted to spaces, but tabs inside text are preserved. Whitespace leading and trailing one-line docstrings is removed.

Source: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#code-style